### PR TITLE
Use Stream 8 image as CentOS 8 is EOL

### PR DESCRIPTION
**Pull request by upadhyeammit**

### CODE REVIEW

**Comment by upadhyeammit** [comment]
Considering we are going to build on RHEL now, I think we can use ubi here: https://www.redhat.com/en/blog/introducing-red-hat-universal-base-image .

@ekohl request to review

**Comment by evgeni** [comment]
Why not the Stream 8 image?

**Comment by upadhyeammit** [comment]
> Why not the Stream 8 image?

I thought ubi is close to building on RHEL, however Stream 8 works as well.

**Comment by evgeni** [comment]
UBI is close to RHEL, yes, but I don't see any benefit for building the installer docs on RHEL

File: scripts/installer/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM centos:8
+FROM centos/centos:stream8
```
Comment (evgeni) [inline scripts/installer/Dockerfile]: Can you please use quay.io/centos/centos:stream8 so it always pulls from quay and not whatever is the default on the machine it's executed on?

Comment (upadhyeammit) [reply scripts/installer/Dockerfile]: done!

**Comment by evgeni** [comment]
I'm cool with that change, leaving final review to Ewoud

**Review by ekohl — APPROVED** [review]
Thanks!

**Pull request merged**